### DISCUSSION TITLE
How to use useBlogPost hook?

### DISCUSSION THREAD

**Discussion by shuashuai** [refinedev/refine#4460] [Q&A]
version: 2.4.0
I am customizing the blog details component blogPostComponent, when I refer to BlogPostProvider, useBlogPost hook, ts reports no '/internal',

```
import {
  BlogPostProvider,
  useBlogPost,
} from "@docusaurus/theme-common/internal";
```

And when running yarn start, there will be the following error

![image](https://github.com/refinedev/refine/assets/30591202/1163ba71-bd98-4308-b1b5-5f07e06b63dc)

How should I use it correctly?

**Comment by BatuhanW** [comment]
Hello @shuashuai I'm not sure if this issue is related to refine. Am I missing something? Feel free to re-open it if you need.
